src/content/content_manager.cc

```diff
@@ -1151,7 +1151,9 @@ std::pair<int, bool> ContentManager::addContainerChain(const std::string& chain,
 
     constexpr auto unwanted = std::array { M_DESCRIPTION, M_TITLE, M_TRACKNUMBER, M_ARTIST }; // not wanted for container!
     for (auto&& unw : unwanted) {
-        lastMetadata.erase(std::remove_if(lastMetadata.begin(), lastMetadata.end(), [=](auto&& m) { return m.first == MetadataHandler::getMetaFieldName(unw); }));
+        auto unwIt = std::remove_if(lastMetadata.begin(), lastMetadata.end(), [=](auto&& m) { return m.first == MetadataHandler::getMetaFieldName(unw); });
```

Comment:
looks like wrong erase usage. iterator parameter is missing.

Reply:
```
std::vector<T,Allocator>::erase

iterator erase( iterator pos );
```

Looks fine to me